src/devices/RGBLedMatrix/RgbLedMatrix.cs

```diff
@@ -386,7 +385,7 @@ public void StartRendering()
         /// <param name="y">Upper left y coordinate on the display</param>
         /// <param name="bitmap">System.Drawing.Bitmap object to draw</param>
         /// <param name="backBuffer">true if want use back buffer, false otherwise</param>
-        public unsafe void DrawBitmap(int x, int y, Bitmap bitmap, bool backBuffer = false)
+        public unsafe void DrawBitmap(int x, int y, BitmapImage bitmap, bool backBuffer = false)
```

Comment:
This is probably the most delicate method as it is unsafe.
I guess you tested this enough :)

Reply:
Actually, I don't have this hardware to test it, but the change is actually quite minor. And I now removed the `unsafe`, since it's no longer required (because we're using Span<byte>)

Reply:
Do you mean you removed the `unsafe` keyword from your next PR?

Reply:
I comited it already here